.github/workflows/assign.yml

```diff
@@ -20,5 +20,8 @@ jobs:
     steps:
       - run: |
           echo "Assigning issue ${{ github.event.issue.number }} to ${{ github.event.comment.user.login }}"
-          gh issue edit ${{ github.event.issue.number }} --add-assignee ${{ github.event.comment.user.login }}
-          gh issue edit ${{ github.event.issue.number }} --remove-label "help wanted"
+          gh issue edit $ISSUE --add-assignee ${{ github.event.comment.user.login }}
+          gh issue edit $ISSUE --remove-label "help wanted"
+        env:
+          GH_TOKEN: ${{ github.token }}
+          ISSUE: ${{ github.event.issue.html_url }}
```

Comment:
`html_url` is required because the repo is not checked in, so `gh` needs to the full url to point to the issue.